File: Weather-app/Models/DailyWeatherData.m
//
//  DailyWeatherData.m
//  Weather-app
//
//  Created by Reve on 11/14/22.
//

#import "DailyWeatherData.h"

@implementation DailyWeatherData

-(instancetype _Nullable) initWithDictionary:(NSDictionary* _Nullable) dict {
    if(self = [super init]) {
        self.time = dict[@"time"];
        self.temperature_2m_min = dict[@"temperature_2m_min"];
        self.temperature_2m_max = dict[@"temperature_2m_max"];
    }
    return self;
}

@end

File: Weather-app/Models/DailyWeatherData.swift
//
//  DailyWeatherData.swift
//  Weather-app
//
//  Created by Reve on 11/20/22.
//

import Foundation

class DailyWeatherData: NSObject, Codable {
    
    let time: [String]
    let temperature_2m_min: [Double]
    let temperature_2m_max: [Double]
}
